modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/list/List.tsx

```diff
@@ -3,12 +3,148 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import ClayAlert from '@clayui/alert';
+import {ClayButtonWithIcon} from '@clayui/button';
+import {Body, Cell, Head, Row, Table} from '@clayui/core';
+import {ClayInput} from '@clayui/form';
+import ClayLayout from '@clayui/layout';
+import classNames from 'classnames';
+import {openModal} from 'frontend-js-web';
 import React from 'react';
 
+import {IFDSViewSectionProps} from '../../../FDSView';
 import {IBaseVisualizationMode} from '../VisualizationModes';
+import {IFDSField} from '../table/Table';
+import AddFieldsModalContent from '../table/modal_content/AddFieldsModalContent';
 
 export interface IList extends IBaseVisualizationMode<'list'> {}
 
-export default function List() {
-	return <>List</>;
+interface IListField {
+	fieldId: string;
+	label: string;
+}
+
+const LIST_VISUALIZATION_MODE_FIELDS: IListField[] = [
+	{fieldId: 'title', label: Liferay.Language.get('title')},
+	{fieldId: 'description', label: Liferay.Language.get('description')},
+	{fieldId: 'symbol', label: Liferay.Language.get('symbol')},
+	{fieldId: 'link', label: Liferay.Language.get('link')},
+	{fieldId: 'label', label: Liferay.Language.get('label')},
+];
+
+const LABEL_CELL_WIDTH = '30%';
+
+export default function List(props: IFDSViewSectionProps) {
+	const [fieldValues, setFieldValues] = React.useState<
+		Record<IListField['fieldId'], IFDSField>
+	>({});
+
+	const onValueChange = (
```

Comment:
[nit] It's a bit misleading to call this `on*`, since it's not a handler, but a convenient util function. Maybe something like `updateFieldValues`? Alternatively, `setFieldValues` can just be inline.

Reply:
I'm in the `on*` team as well :slightly_smiling_face: :+1: